src/peer/room.js

```diff
@@ -60,6 +62,30 @@ class Room extends EventEmitter {
     this._pcConfig = this._options.pcConfig;
   }
 
+  /**
+   * Check whether the size of data to send is over the limits or not.
+   * @param {object} data - The data to check.
+   */
+  isOverLimits(data) {
```

Comment:
I think it is better to change the function name. The subject of `isOverLimits` should not be` Room`.